doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -278,6 +278,12 @@ also use the ``/usr/libexec/suricata/ebpf/xdp_filter.bpf`` (in our example TCP o
     # if the ebpf filter implements a bypass function, you can set 'bypass' to
     # yes and benefit from these feature
     bypass: yes
+    # if the ebpf filter implements ERSPAN2 bypass function,
+    # you can set 'bypass-erspan2' to yes and benefit from this feature
```

Comment:
I'm missing an explanation of why a user would set this and what the effect is

Reply:
removed in next version